src/GameLoop.fs

```diff
@@ -123,7 +140,8 @@ type GameLoop (config: GameConfig) as this =
         // by default, all sprites are drawing on .End() in a batch
         // immediate changes this so they are drawn as called, which allows us to
         // change the sampler state (e.g. for pixel graphics vs text) between different sprite calls
-        spriteBatch.Begin (sortMode = SpriteSortMode.Immediate)
+        spriteBatch.Begin (sortMode = SpriteSortMode.Immediate, 
+                            samplerState = SamplerState.PointClamp)
```

Comment:
At present, the use of point clamp vs linear clamp can be controlled via the setPixelSampling and setSmoothSampling viewables. Any reason why you want to fix it? Probably works better as a default

Reply:
also i posted a small change just changing soem comments. not sure how that gets into here

Reply:
Default is good too. 
The idea is to make the `begin` function settable, such that users can custom it.
(like camera 2d, or maybe we can just put the camera in?)

Reply:
> At present, the use of point clamp vs linear clamp can be controlled via the setPixelSampling and setSmoothSampling viewables. Any reason why you want to fix it? Probably works better as a default

ok. i didnt noticed that before.

Reply:
one of the reasons it can be toggled during a scene is that text tends to look a lot better with linear clamp, while textures and shapes better with point

Reply:
In my trial branch, I used a third-party C# lib called `Myra` to render UI instead of using Xelmish because it is more complete. (add some type extensions to make it more declarative)

I do ignore the difference between the linear clamp and the point clamp.

Reply:
Oh yep, how did that work with the elmish model?

Reply:
If you're interested in it I'll leave a link. 
https://github.com/RickyYChan/Xelmish/tree/ricky-trial

The main idea is to make it more declarative. 
I use some type-extensions so users can just set everything. 

```fs
let ui model dispatch ast =
        desktop (
            Root =
                horizontalStackPanel (
                    Id = "mainGrid",
                    Visible = true,
                    Spacing = 10,
                    Padding = thick 10,
                    children =
                        [ textButton (
                            Id = "tbtn",
                            Text = "Hello world!",
                            GridColumn = 0,
                            command = (fun _ -> -10 |> Horizontal |> dispatch)
                          )
                          label (Text = "Hello myra in xelmish!", GridColumn = 1) ]
                )
        )
```

And I think we should use mutable way instead of re-construct it since the lib is built that way. 
I'll use 
```fs
let uisets (model: Model) ast =
        [ SetUI("tbtn",
              fun ctrl -> 
                (ctrl :?> textButton).Visible <- 
                    if model.UI.IsPressed then
                        false
                    else
                        true
          )
        // if a widget is not visible, then the stackPanel will not save its position. Nice ~
         ]
```
(I also tried reflection to let it more like bindings or something, it seems like `SetUI(typeof<_>, elementId, nameofProperty, newValue)`, and i donnot think this is something good)

in Xelmish.Program, we get 
```fs
    let setState model dispatch =
// ...

        if loop.CanInitUI() then
            loop.Desk <- lazy (ui model dispatch loop.Ast) // model = initModel.
        // don't reconstruct desktop 'cause we use mutable way.

// ...

        if loop.Desk.IsValueCreated // if game is running ...
        then
            uisets model loop.Ast
            |> List.iter (function
                | SetUI (id, act) -> id |> loop.Desk.Value.GetWidgetByID |> act)

        loop.View <- (program |> Program.view) model dispatch
```
The way of initializing UI is good, but I think the mutable way is not that elmish.

Reply:
Cool. I'll take a proper look - UI is always the toughest bit for me. Will merge this PR anyway